src/styles/SearchResults.css

```diff
@@ -0,0 +1,34 @@
+.search-results {
+  margin-top: 30px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.search-results__image {
+  width: 80%;
+  height: 200px;
+  border: 3px solid red;
+  margin-top: 5px;
+  cursor: pointer;
+}
+
+@media screen and (min-width: 768px) {
+  .search-results__image {
+    margin: 5px;
+    height: 500px;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .search-results__image {
+    margin: 5px;
```

Comment:
The `margin` property here isnt needed, as nothing is changing from the declaration on line 18. Is it deliberate to add a margin to all sides in the above rule or should it just specify `margin-top`?